libs/platform/ebpf_object.h

```diff
@@ -22,6 +22,8 @@ extern "C"
     typedef struct _ebpf_object ebpf_object_t;
     typedef void (*ebpf_free_object_t)(ebpf_object_t* object);
 
+    // This type probably ought to be renamed to avoid confusion with
+    // ebpf_object_t in libs\api\api_internal.h
```

Comment:
I think Alan had once suggested that we can change the struct names in EC to ebpf_core_object, ebpf_core_program, etc.

Reply:
Sure.  But left to a separate PR since not related to removing duplicate public APIs where standard public bpf apis exist